components/org.wso2.carbon.identity.local.auth.push.authenticator/src/main/java/org/wso2/carbon/identity/local/auth/push/authenticator/PushAuthContextManager.java

```diff
@@ -0,0 +1,32 @@
+package org.wso2.carbon.identity.local.auth.push.authenticator;
+
+import org.wso2.carbon.identity.local.auth.push.authenticator.model.PushAuthContext;
+
+/**
+ * This interface manages push auth context operations for push based authentication.
+ */
+public interface PushAuthContextManager {
+
+    /**
+     * Store push authentication context.
```

Comment:
Mention where this context is stored. i.e. this is persisted in the temp session store.